frappe/__init__.py

```diff
@@ -89,6 +89,27 @@
 	warnings.simplefilter("always", PendingDeprecationWarning)
 
 
+def _get_local_proxy(self: Local, name: str) -> LocalProxy:
+	"""Get local proxy object by name."""
+
+	_local_contextvar = self._Local__storage
```

Comment:
Based on what I read from their code and docs: Their implementation is the way it is for nested contexts. I don't know any such usage in our code base, so it _should_ be just fine.

---

We should pin `werkzeug` with `==` if we are gonna touch internals.

Reply:
we can also pass our own contextvar to `werkzeug.Local` constructor to avoid this line.